pwnlib/elf/elf.py

```diff
@@ -1328,13 +1328,23 @@ def relro(self):
         .. _DF_BIND_NOW: http://refspecs.linuxbase.org/elf/gabi4+/ch5.dynamic.html#df_bind_now
 
         """
+        if not any('GNU_RELRO' in str(s.header.p_type) for s in self.segments):
```

Comment:
"Partial RELRO" is consistent with checksec.sh, I do not think it should be changed, it would probably just confuse people.